frontend/internal-packages/agent/src/repositories/supabase.ts

```diff
@@ -331,23 +244,28 @@ export class SupabaseSchemaRepository implements SchemaRepository {
     }
 
     const patchArrayHistory = previousVersions
-      ?.map((version) => {
+      .map((version) => {
         const parsed = v.safeParse(operationsSchema, version.patch)
         if (parsed.success) {
           return parsed.output
         }
         return null
       })
-      .filter((version) => version !== null)
+      .filter((patch): patch is NonNullable<typeof patch> => patch !== null)
```

Comment:
[ask]
This may be a little unrelated to this development, but would you revert to not null where the `building_schema_versions` `patches` and `reverse_patches` are currently nullable?

https://github.com/liam-hq/liam/blob/main/frontend/internal-packages/db/supabase/migrations/20250715070104_make_building_schema_versions_patch_nullable.sql

Reply:
Thank you so much!🌟